README.md

```diff
@@ -1 +1,2 @@
-# code-Review
\ No newline at end of file
+# code-Review
+ this project is made by Alelign Ayana
```

Comment:
add something that describes your code

Reply:
- [ ] @atatm  kindly try always to include an explanation, a "why", for each requested change. For example:

> - **_This is a well-explained request_**: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
> - _**This request lacks a "why":**_ Why don't you use an article tag here instead of a div?